build/lib/helm_require.sh

```diff
@@ -48,6 +48,8 @@ export IMAGE_TAG
 export HELM_REGISTRY
 envsubst <helm/sedfile.template >${SEDFILE}
 echo "s,version: v,version: ,g" >>${SEDFILE}
+echo "/^version:/s,-helm,," >>${SEDFILE}
+echo "/^version:/s,-,+," >>${SEDFILE}
```

Comment:
We might be leaving the `+` for dev builds.

Reply:
Yes, this isn't removing the `+`, it's changing the `-` to a `+`, so that it's a valid semver.

Reply:
I feel like you didn't phrase that right because `-` is valid for semver. You mean if we use the `-` it is ignored by semver, right? I'm just not 100% sure how the production builds will work and I'm not 100% sure 0.1.0+1 is seen by helm has an older version than 0.1.0+2. I was concerned we'd have to change the helm release scheme to use 0.1.1 and 0.1.2 which will work for sure

Reply:
`-` is for pre-release, it will be ignored by anything that excludes pre-release. We're putting the git commit id of the repo, so `+` (build metadata) seems more appropriate.

Reply:
We should be using 0.1.1, 0.1.2 as release scheme, disjoint from the appVersion. This seems to be the standard anyways.